react/components/EXPERIMENTAL_Table/README.md

```diff
@@ -642,6 +642,1091 @@ function VisibilityExample() {
+- A wrapper around `InputSearch` component. The props are the same.
+
+#### InputAutocomplete
+
+- A wrapper around `AutocompleteInput` component. The props are the same.
+
+#### Button Group
+
+Represents the group of buttons located at the right. It has other composites that are described down below.
+
+##### Columns
+
+- A button that toggles columns visibility.
+- It is recommended to combine it with the `useTableVisiblitity` hook.
+
+```ts
+enum Alignment {
+  Left = 'left',
+  Right = 'right',
+}
+```
+
+| Property     | Type                         | Required | Default | Description                      |
+| ------------ | ---------------------------- | -------- | ------- | -------------------------------- |
+| label        | string                       | ✅       | 🚫      | General label                    |
+| showAllLabel | string                       | ✅       | 🚫      | Label for the show all button    |
+| hideAllLabel | string                       | ✅       | 🚫      | Label for the hide all button    |
+| visibility   | Return of useTableVisibility | ✅       | 🚫      | Visibility of the columns        |
+| alignMenu    | Alignment                    | 🚫       | 🚫      | Menu alignment                   |
+| disabled     | boolean                      | 🚫       | false   | If the button is disabled or not |
+
+##### Density
+
+- A button that changes the row's density.
+- It is recommended to combine it with the `useTableMeasures` hook.
+
+```ts
+enum Alignment {
+  Left = 'left',
+  Right = 'right',
+}
+
+enum Density {
+  Compact = 'compact',
+  Regular = 'regular',
+  Comfortable = 'comfortable',
+}
+```
+
+| Property         | Type                       | Required | Default | Description                      |
+| ---------------- | -------------------------- | -------- | ------- | -------------------------------- |
+| density          | Return of useTableMeasures | ✅       | 🚫      | Density object                   |
+| compactLabel     | string                     | ✅       | 🚫      | Label of the compact option      |
+| regularLabel     | string                     | ✅       | 🚫      | Label of the regular option      |
+| comfortableLabel | string                     | ✅       | 🚫      | Label of the comfortable option  |
+| alignMenu        | Alignment                  | 🚫       | 🚫      | Menu alignment                   |
+| handleCallback   | (density: Density) => void | 🚫       | 🚫      | Triggered on change density      |
+| disabeld         | boolean                    | 🚫       | false   | If the button is disabled or not |
+
+##### Download
+
+- Button to handle download or export actions.
+
+| Property | Type       | Required | Default | Description                      |
+| -------- | ---------- | -------- | ------- | -------------------------------- |
+| onClick  | () => void | ✅       | 🚫      | Action on click button           |
+| label    | string     | 🚫       | 🚫      | Button text                      |
```

Comment:
If its not required it should have a default, even it it is '', no?

Reply:
yep, nice catch